site/docs/components/multiline-input/usage.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### When to use Multiline Input
+
+A Multiline Input is recommended when a user is asked a subjective question and the length of their answer is unknown. The input can give them plenty of space to answer the question and review the answer. Set the height accordingly if you can take an educated guess at the possible length of their answer–see the [Number of Rows example](../examples#number-of-rows).
```

Comment:
the link should be `../multiline-input/examples#number-of-rows` but it doesn't work in the current set up sadly, we can probably remove it

Reply:
will rm for now